docs/sources/get-started/component_controller.md

```diff
@@ -102,7 +102,7 @@ If this address collides with a real target on your network, change it to someth
 Components must opt-in to using in-memory traffic.
 Refer to the individual documentation for components to learn if in-memory traffic is supported.
 
-## Updating the configuration file
+## Update the configuration file
```

Comment:
Hmmm you made me read the topic section (instead of myopically focusing on just the linting warnings)  and... the title is better rewritten as `Reload the configuration file`

Reply:
Hmmm reading it again.. yeah I see the point.. no task. Going with your suggestion